src/templating/native/nativeTemplateEngine.js

```diff
@@ -10,7 +13,16 @@ ko.nativeTemplateEngine.prototype['renderTemplateSource'] = function (templateSo
         templateNodes = templateNodesFunc ? templateSource['nodes']() : null;
 
     if (templateNodes) {
```

Comment:
Not unless we refactored some other code. Notice the Array check inside and handling if it's not an array. The only way that change would make sense would be if we inserted an `else if (templateNodes)` before the `else`.